Packs/Kafka/Integrations/KafkaV3/README.md

```diff
@@ -0,0 +1,156 @@
+
+This integration was integrated and tested with version 2.7.1 of Kafka.
+
+This integration is fully compatible with the Kafka v2 integration.
+
+## Configure Kafka v3 on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Instances**.
+2. Search for Kafka v3.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | CSV list of Kafka brokers to connect to, e.g. 172.16.20.207:9092,172.16.20.234:9093 |  | True |
+    | Use TLS for connection |  | False |
+    | Trust any certificate (not secure) |  | False |
+    | CA certificate of Kafka server (.cer) |  | False |
+    | Client certificate (.cer) |  | False |
+    | Client certificate key (.key) |  | False |
+    | Client certificate key password (if required) |  | False |
+    | Password |  | False |
+    | Topic to fetch incidents from (Required for fetch incidents) |  | False |
+    | CSV list of partitions to fetch messages from |  | False |
+    | Offset to fetch messages from (exclusive) | The initial offset to start fetching from, not including the value set \(e.g. if 3 is set, the first event that will be fetched will be with offset 4\). If you want to start from the earliest or latest, type in 'earliest' or 'latest' accordingly. | False |
+    | Max number of messages to fetch |  | False |
+    | Fetch incidents |  | False |
+    | Incident type |  | False |
+    | Max number of bytes per message | The max number of message bytes to retrieve in each attempted fetch request. Should be in multiples of 1024. If the fetching process takes a long time, consider increasing this value. Default is '1048576'. | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### kafka-print-topics
+***
+Prints all topics and their partitions.
+
+
+#### Base Command
+
+`kafka-print-topics`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| include_offsets | Whether to include the first and last offset for a topic, when printing a list of topics and partitions. Possible values are: true, false. Default is true. | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Kafka.Topic.Name | String | Kafka topic name | 
+| Kafka.Topic.Partitions.ID | Number | Topic partition ID | 
+| Kafka.Topic.Partitions.EarliestOffset | Number | Topic partition earliest offset | 
+| Kafka.Topic.Partitions.LatestOffset | Number | Topic partition latest offset | 
+
+
+#### Command Example
+```!kafka-print-topics```
+
+#### Human Readable Output
+
+![](https://raw.githubusercontent.com/demisto/content/master/docs/images/Integrations/Kafka_V2_mceclip3.png)
```

Comment:
can we please not use images, but markdown ?

also, missing context output example